src/Compilers/VisualBasic/Portable/VisualBasicParseOptions.vb

```diff
@@ -19,6 +19,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Private _features As ImmutableDictionary(Of String, String)
 
         Private _preprocessorSymbols As ImmutableArray(Of KeyValuePair(Of String, Object))
+        Private _specifiedLanguageVersion As LanguageVersion
         Private _languageVersion As LanguageVersion
```

Comment:
The only places where they being **assigned** a value is from within this class.